lib/src/over_react_redux/over_react_flux.dart

```diff
@@ -355,7 +355,7 @@ UiFactory<TProps> Function(UiFactory<TProps>) connectFlux<TStore extends flux.St
   );
 }
 
-bool _shallowMapEquality(Map a, Map b) => const MapEquality().equals(a, b);
+bool _shallowMapIdentityEquality(Map a, Map b) => const MapEquality(keys: IdentityEquality(), values: IdentityEquality()).equals(a, b);
```

Comment:
thoughts on making this (and `_shallowMapEquality`) public and importable?
In situations where I'ld like to configure my connect to use the `==` based equality, i'll have to create my own MapEquality.

It is nbd if we keep it private, it is obviously not hard to make my own, but i'm just thinkin it might be a nice to have

Reply:
Good point, may as well. 

There's also a little more optimal implementation of this that I should probably switch over to using. (MapEquality is surprisingly inefficient)